src/app/core/state/food/food.selectors.ts

```diff
@@ -0,0 +1,22 @@
+import { AppState } from '../index';
+
+import { createSelector } from '@ngrx/store';
+
+const selectAppState = (state: AppState) => state;
+
+export const foodSelectors = {
+    counts: createSelector(selectAppState, (state: AppState) => {
+        let totalCount = 0;
+        let bringingCount = 0;
+
+        state.food.food.forEach((food) => {
+            totalCount += food.count;
+            bringingCount += food.users.length;
+        });
+
+        return {
```

Comment:
This makes way more since as a single selector than multiple. Nice!

I wonder if you should also add the percentage complete too instead of in the overview component.